### PR TITLE
- Add Colibri Race md file

### PR DESCRIPTION
**Pull request by larryho5**

### CODE REVIEW

**Comment by thenickdude** [comment]
Looks good to me!

File: docs/Board - ColibriRace.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	| 2   | PWM2           | MOTOR 2                		               |
+	| 3   | PWM3           | MOTOR 3                                      |
+	| 4   | PWM4           | MOTOR 4                                      | 
+	| 5   | PWM5 	       | MOTOR 5 (For Y6 or Hex X)                     | 
+	| 6   | PWM6 	       | MOTOR 6 (For Y6 or Hex X)                     | 
+	| 7   | BST SDA        | Use For TBS CorePro Control Device           |
+	| 8   | BST SCL        | Use For TBS CorePro Control Device           |
+	| 9   | PWM7           | Can be a normal GPIO (PA2) or PWM            |
+	| 10  | PWM8           | Can be a normal GPIO (PA2) or PWM            |
+	| 11  | 12.2V DCDC     | If 12v is detected, the Blue LED will trun on|
```
Comment (Echelon9) [inline docs/Board - ColibriRace.md]: s/trun/turn

File: docs/Board - ColibriRace.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	| 2   | Ground            |                                              |
+	| 3   | UART3 TX          | GPS                                          |
+	| 4   | UART3 RX          | GPS                                          |
+	| 5   | SDA               | mag, pressure, or other i2c device           |
+	| 6   | SCL               | mag, pressure, or other i2c device           |
+
+### IO_3 - RC input
+
+	When RX_PPM/RX_SERIAL is used the IO_3 pinout is as follows.
+	if RX_SERIAL, Telemetry need to set to UART2 and choose your device.
```
Comment (Echelon9) [inline docs/Board - ColibriRace.md]: These two sentences could possibly be reworked so the grammar and intention is clearer.

**Comment by larryho5** [comment]
If no more problem, please accept this PR. Thank You. :)

File: docs/Board - ColibriRace.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+### IO_5 - Free UART
+
+	| Pin | Function          | Notes                                        |
+	| --- | ----------------- | -------------------------------------------- |
+	| 1   | UART1 TX          | Free UART                                    |
+	| 2   | UART1 RX          | Free UART                                    |
+	| 3   | Ground            |                                              |
+	| 4   | VCC_OUT           | 4.7 Volt output to the device                |
+
+### IO_6 - IR TX (externsion)
```
Comment (atomiclama) [inline docs/Board - ColibriRace.md]: Nit picking here but spelling incorrect for extension

**Comment by larryho5** [comment]
Fixed, Thank you :)

File: docs/Board - ColibriRace.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+	| 1   | PWM1           | MOTOR 1                                      |
+	| 2   | PWM2           | MOTOR 2                                      |
+	| 3   | PWM3           | MOTOR 3                                      |
+	| 4   | PWM4           | MOTOR 4                                      |
+	| 5   | PWM5           | MOTOR 5 (For Y6 or Hex X)                    |
+	| 6   | PWM6           | MOTOR 6 (For Y6 or Hex X)                    |
+	| 7   | BST SDA        | Use For TBS CorePro Control Device           |
+	| 8   | BST SCL        | Use For TBS CorePro Control Device           |
+	| 9   | PWM7           | Can be a normal GPIO (PA2) or PWM            |
+	| 10  | PWM8           | Can be a normal GPIO (PA2) or PWM            |
```
Comment (hydra) [inline docs/Board - ColibriRace.md]: PA2 is duplicated here.

File: docs/Board - ColibriRace.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+	| 1     | VCP          | USB PORT       | Main Port For MSP                         |
+	| 2     | USART1       | FREE PORT      | PC4 and PC5(Tx and Rx respectively)       |
+	| 3     | USART2       | PPM Serial     | PA15                                      |
+	| 4     | USART3       | GPS PORT       | PB10 and PB11(Tx and Rx respectively)     |
+
+## Pinouts
+
+	Full pinout details are available in the manual, here:
+
+	http://www.team-blacksheep.com/colibri_race
```
Comment (hydra) [inline docs/Board - ColibriRace.md]: i tried this link but it redirected to the home page.

**Comment by hydra** [comment]
Going to merge this now, please create a new PR for the update to https://github.com/cleanflight/cleanflight/pull/1279#discussion_r38812699

File: docs/Board - ColibriRace.md (hunk 5 above)
Comment (tbs-trappy) [inline docs/Board - ColibriRace.md]: link will work once the product is live. it's not yet available to end-customers. we are still waiting for it get ready for shipping :)

**Pull request merged**